pkg/api/index.go

```diff
@@ -50,16 +51,20 @@ func (hs *HTTPServer) setIndexViewData(c *contextmodel.ReqContext) (*dtos.IndexV
 	// Locale is used for some number and date/time formatting, whereas language is used just for
 	// translating words in the interface
 	acceptLangHeader := c.Req.Header.Get("Accept-Language")
-	locale := "en-US"
-	language := "" // frontend will set the default language
+	locale := "en-US" // frontend will set the default locale, but we set English as default
+	language := ""    // frontend will set the default language
 
 	if prefs.JSONData.Language != "" {
 		language = prefs.JSONData.Language
 	}
-
-	if len(acceptLangHeader) > 0 {
-		parts := strings.Split(acceptLangHeader, ",")
-		locale = parts[0]
+	var ctx context.Context
```

Comment:
Don't create a new empty context - there should be an existing one to grab somewhere...

Reply:
I think 

```go
if hs.Features.IsEnabled(c.Req.Context(), featuremgmt.FlagLocaleFormatPreference) {
```

Is what you want.